### ISSUE TITLE
Feature Request: RFID auth notification/state

### ISSUE THREAD

**Issue by StefanTenb** [iobroker-community-adapters/ioBroker.kecontact#126]
Hi, 

as this adapter is working quite well, I wanted to do some more advanced automation with it. 

I own a P30 with RFID auth feature and would like to know if there is a way to see a successfull auth in one of the adapters variables. The P30 gives a positiv "beep" and a blue led in that case, but I couldn´t see any changes in the states. 

What i want to do: 
1. Auth with RFID at the P30
2. Detect this event with io broker
3. Send a signal to my car to unlock the charging port
4. Plug the charger into the vehicle 

It seems that the states in the P30 are only updated if a vehicle is already plugged and also the sessions are not started by the RFID auth. 

From the UDP Programmers Guide by Keba, I thought that "Enable sys" (enableSys) would provide this auth state.

**Comment by StefanTenb** [comment]
I did some more tests and checked the adapter in debug logging. 

Within the log I could see that "report 2" also includes the variables "authON" and "authReq", and the authReq is indeed changing if I present the RFID tag to the P30. I have to apologize, seems that I used an older version of the UDP programers guide, the new one includes the RFID variables with descriptions.

So my only request for a change would be the comment of the variable: 

Currently it says: _State of Authorization via RFID_ 
Alternative: _Authorization via RFID required_

The manual defines: 
False: _Authorization via RFID card is not required. **OR** The authorization via RFID card was already
performed._
True: _Authorization via RFID card is required._ 

With  _State of Authorization via RFID_ = _True_ I would assume that the authorization is given. 

Thank you very much for the very fast reply and in general for the work on the adapter.
